c7n/resources/s3.py

```diff
@@ -110,14 +110,17 @@ def load_resource(self, item):
         # owner is under acl per describe
         resource.pop('Owner', None)
 
+        missing = object()
         for k, null_value in S3_CONFIG_SUPPLEMENT_NULL_MAP.items():
             if cfg.get(k) == null_value:
                 continue
             method = getattr(self, "handle_%s" % k, None)
             if method is None:
                 raise ValueError("unhandled supplementary config %s", k)
                 continue
-            v = cfg[k]
+            v = cfg.get(k, missing)
```

Comment:
nit: maybe worth plopping this line at the top of the loop to avoid having two subtly different `cfg.get()` calls

Reply:
simplified by just doing k in cfg check at the top